### DISCUSSION TITLE
network disconnected

### DISCUSSION THREAD

**Discussion by vusinayes** [iotaledger/firefly#2269] [Help & Support] [answered]
when i open my wallet, it comes up "something went wrong", & it says network disconnected. i don't see my balance & can't stake. can somebody help me?

**Comment by MiKeF05** [comment]
Can you please share the error log of Firefly? Do you use a VPN or special firewall?

**Comment by vusinayes** [comment]
Wed, 16 Feb 2022 22:46:13 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:46:13 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:46:03 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:46:03 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:53 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:53 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:43 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:43 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:39 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/events): error
trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

Wed, 16 Feb 2022 22:45:33 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/addresses/iota1qz0eq8rrarm6367cgejxjxu85ws75qpe0xqzxh9gh5pl4xh6pm2a7xsu7h2/outputs?include-spent=true):
error trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:32 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz0eq8rrarm6367cgejxjxu85ws75qpe0xqzxh9gh5pl4xh6pm2a7xsu7h2):
error trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

Wed, 16 Feb 2022 22:45:32 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 16 Feb 2022 22:45:32 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/events): error
trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

Wed, 26 Jan 2022 00:31:54 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz0eq8rrarm6367cgejxjxu85ws75qpe0xqzxh9gh5pl4xh6pm2a7xsu7h2):
error trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

Wed, 26 Jan 2022 00:31:54 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 26 Jan 2022 00:31:54 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/addresses/iota1qz0eq8rrarm6367cgejxjxu85ws75qpe0xqzxh9gh5pl4xh6pm2a7xsu7h2/outputs?include-spent=true):
error trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer`

Wed, 26 Jan 2022 00:31:54 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 26 Jan 2022 00:31:54 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/events): error
trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

Wed, 12 Jan 2022 01:24:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:12 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:24:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:22 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:23:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:52 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:42 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:32 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:12 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:22:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:42 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:02 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:21:02 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:42 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:32 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:32 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:22 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:22 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:12 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:12 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:03 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:20:03 GMT
ClientError: `error sending request for url (
https://mainnet-node.tanglebay.com/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:19:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/addresses/iota1qz0eq8rrarm6367cgejxjxu85ws75qpe0xqzxh9gh5pl4xh6pm2a7xsu7h2/outputs):
error trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:19:52 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz0eq8rrarm6367cgejxjxu85ws75qpe0xqzxh9gh5pl4xh6pm2a7xsu7h2):
error trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

Wed, 12 Jan 2022 01:19:52 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.cafe/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:19:51 GMT
ClientError: `error sending request for url (
https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect:
invalid peer certificate contents: invalid peer certificate: UnknownIssuer`

Wed, 12 Jan 2022 01:19:51 GMT
ReqwestError: error sending request for url (
https://chrysalis-nodes.iota.cafe/api/plugins/participation/events): error
trying to connect: invalid peer certificate contents: invalid peer
certificate: UnknownIssuer

**Reply by MiKeF05** [reply]
Do you use a work PC?

**Comment by vusinayes** [comment, marked as answer]
thanks for the response, i use a filter https://www.techloq.com/

**Comment by vusinayes** [comment]
I use a pc with aforementioned filter. Is this the problem? is there a way
to solve this?

**Reply by MiKeF05** [reply]
It seems like that this filter blocks the connection to the node. Can you try to temporarily disable this filter? Or maybe add that the node connection doesn't get blocked.

**Comment by vusinayes** [comment]
i'll contact them. How should I describe them the "node connection''?
[tha're not in crypto...]
such a pity all the staking rewards i didn't make

**Reply by MiKeF05** [reply]
When you look at the error log of Firefly you see a URL. This URL is the connection to the node. And it needs to be allowed.

**Comment by vusinayes** [comment]
Thanks!

**Comment by vusinayes** [comment]
it doesn't look like the filter is the problem. What does that mean?

**Comment by vusinayes** [comment]
hi. The filter company claims that they opened the links. but it still
doesn't work. the TanglePay extension works, so it can't be that the nodes
are the problem. I still haven't received a response to my previous email.
thanks
